### PR TITLE
Fixed schema types

### PR DESCRIPTION
**Pull request by gleonard-m**
Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

### CODE REVIEW

File: sql/moz-fx-data-marketing-prod/ga_derived/downloads_with_attribution_v1/schema.yaml [hunk 1]
```diff
@@ -129,26 +129,25 @@ fields:
   mode: NULLABLE
   name: count_dltoken_duplicates
   type: INTEGER
-- description: "This column represents if there was at least one other download completed during the same\
-    \ session as this download. "
+- description: 'This column represents if there was at least one other download completed
```
Comment (lucia-vargas-a) [inline sql/moz-fx-data-marketing-prod/ga_derived/downloads_with_attribution_v1/schema.yaml:132]: For the column descriptions you shouldn't need the quotes:
```suggestion
- description: Indicates if at least one more download was completed during the
    same session of the current download.
```

Comment (gleonard-m) [reply sql/moz-fx-data-marketing-prod/ga_derived/downloads_with_attribution_v1/schema.yaml:132]: The quotes and formatting is odd, running `schema update` seems to add them.

Comment (lucia-vargas-a) [reply sql/moz-fx-data-marketing-prod/ga_derived/downloads_with_attribution_v1/schema.yaml:132]: Ah, pobably a recent change. I normally don't use them, like [in the aggregates](https://github.com/mozilla/bigquery-etl/blob/b5facd7b69fa9c8ec1e735e7e81ee27651abb07b/sql/moz-fx-data-shared-prod/telemetry_derived/active_users_aggregates_v1/schema.yaml#L86-L87), and I find the query much easier to read.

**Review by lucia-vargas-a — APPROVED** [review]

File: sql/moz-fx-data-marketing-prod/ga_derived/downloads_with_attribution_v1/metadata.yaml [hunk 2]
```diff
@@ -26,20 +26,23 @@ description: |-
     * count_dltoken_duplicates will be >= 1.
     * A value of 0 means there are no duplicates (expected state).
     * A value of 1 means there was 1 other row with the dltoken value, etc.
-
 owners:
 - gleonard@mozilla.com
 labels:
   incremental: true
   schedule: daily
+  owner1: gleonard
```
Comment (ANich) [inline sql/moz-fx-data-marketing-prod/ga_derived/downloads_with_attribution_v1/metadata.yaml:34]: IIRC owner labels are auto-generated

File: sql/moz-fx-data-marketing-prod/ga_derived/downloads_with_attribution_v1/schema.yaml (hunk 1 above)
Comment (gleonard-m) [reply sql/moz-fx-data-marketing-prod/ga_derived/downloads_with_attribution_v1/schema.yaml]: I removed them all the reran schema update, the quotes were not added back.

File: sql/moz-fx-data-marketing-prod/ga_derived/downloads_with_attribution_v1/metadata.yaml (hunk 2 above)
Comment (ANich) [reply sql/moz-fx-data-marketing-prod/ga_derived/downloads_with_attribution_v1/metadata.yaml:34]: Gotcha. Not sure if this is the intended behaviour but maybe it is cc @wwyc

**Pull request merged**